pkg/spec/spec.go

```diff
@@ -37,7 +37,7 @@ type ServicePortMod struct {
 type ServiceSpecMod struct {
 	api_v1.ServiceSpec `json:",inline"`
 	Name               string           `json:"name,omitempty"`
-	Ports              []ServicePortMod `json:"ports"`
+	Ports              []ServicePortMod `json:"ports,conflicting"`
```

Comment:
Above every conflicting field? I'd rather want this to be a part of the developer docs, and the tests are going to fail anyway if a field is not handled and is not marked as "conflicting".